onchain/rollups/contracts/dapp/CartesiDApp.sol

```diff
@@ -133,16 +134,14 @@ contract CartesiDApp is
             revert VoucherReexecutionNotAllowed();
         }
 
-        // execute voucher
-        (bool succ, ) = _destination.call(_payload);
+        // execute voucher, handles success and failure internally.
+        // If `_destination` reverts with a revert reason or custom
+        // error, it is bubbled up by this function.
```

Comment:
Revert reason and custom error are Solidity concepts, but we might be calling contracts not written in Solidity. To adopt a more language-agnostic view, we could simply say:

```suggestion
        // execute voucher
        // propagates any error in case of failure
```

Also, it might be worth to mention that, if the low-level call succeeds, we are discarding the return data entirely. I would like, however, on another PR, to make this function to forward this return data as-is.

Reply:
I will keep that in handles success and failure internally, I'll apply the rest.